src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Helpers/MinIoLinkGenerator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using HashidsNet;
+using Microsoft.Extensions.Options;
+using Smart.FA.Catalog.Infrastructure.Services.Options;
+
+namespace Smart.FA.Catalog.Infrastructure.Helpers;
+
+public class MinIoLinkGenerator : IMinIoLinkGenerator
```

Comment:
Service might be a better name to stay consistent in our naming
```suggestion
public class MinIoLinkService : IMinIoLinkService
```

Reply:
I suggested Victor that name, I think it is good as it makes it clear what it does.
An example:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.passwordhasher-1?view=aspnetcore-6.0

However MinIOService we have does a bunch of different things it makes sense have suffix.

You will also most likely name a service that provides IConnection IConnectionProvider.